wildfire/goes/downloader.py

```diff
@@ -44,3 +47,150 @@ def download_scan(s3_bucket, s3_key, local_directory):
     )
     s3.download_file(Bucket=s3_bucket, Key=s3_key, Filename=local_filepath)
     return local_filepath
+
+
+def download_batch(satellite, regions, channels, start, end, local_directory):
```

Comment:
Why did you back off the ENUMs?

Reply:
Umm I would think about adding back, but I just decided that I would rather underbuild than overbuild right now